relay-event-normalization/src/event.rs

```diff
@@ -504,35 +503,62 @@ fn normalize_device_class(event: &mut Event) {
     }
 }
 
-/// Process the last frame of the stacktrace of the first exception.
+/// Normalizes all the stack traces in the given event.
```

Comment:
so is this really a refactor ? this seems like a functional change

Reply:
Good question! Currently, light normalization normalizes the last frame of the stack trace of the first exception, and the rest of the stack trace is normalized in store processor. This PR unifies both steps, i.e. running normalization on all stack traces directly. You can see this in `normalize/mod.rs:548` below in the diff.

Reply:
ah right i see. So I'm wondering why it was made like that in the first place, the comments said only one was needed for metric extraction but then we normalized all anyway? anyway, LGTM!

Reply:
Only one is required for metrics extraction, so we normalized it before this happened. We still had to normalize all of them, so this normalization remained unchanged in the store processor.